services/frontend/src/App.tsx

```diff
@@ -83,6 +88,19 @@ export default function App() {
     }
   }, [dispatch, isAuthenticated]);
 
+  useEffect(() => {
+    const handler = () => {
+      setViewHeight();
+    };
```

Comment:
You don't really need a function here. You can just use `setViewHeight` directly like this:

```
window.addEventListener("resize", setViewHeight);
...
```